deployments/deployments.json

```diff
@@ -62,6 +62,10 @@
     "UniswapV3_SwapAndBridge": { "address": "0x9220fa27ae680e4e8d9733932128fa73362e0393", "blockNumber": 56675837 },
     "MulticallHandler": { "address": "0x924a9f036260DdD5808007E1AA95f08eD08aA569", "blockNumber": 59194129 }
   },
+  "232": {
+    "SpokePool": { "address": "0xe7cb3e167e7475dE1331Cf6E0CEb187654619E12", "blockNumber": 193290 },
+    "MulticallHandler": { "address": "0xc5939F59b3c9662377DdA53A08D5085b2d52b719", "blockNumber": 193291 }
+  },
```

Comment:
Are we planning on deploying the SpokePoolVerifier?

Reply:
Good catch. I do need to do that still.

Reply:
Nice thanks, I don't think it's super urgent though. 🙏

Reply:
https://github.com/across-protocol/contracts/pull/927/commits/ee3a3f3e2a5a9534f847547844059e2a6bf158b9